### PR TITLE
Add tests for OVN charm NRPE checks

### PR DESCRIPTION
**Pull request by xavpaice**
The ovn-* charms have added NRPE service checks, this ensures that the
service check files have been added to the units as expected.

### CODE REVIEW

File: zaza/openstack/charm_tests/ovn/tests.py [hunk 1]
```diff
@@ -50,6 +62,19 @@ def test_pause_resume(self):
...
+        """Confirm that the NRPE service check files are created."""
+        units = zaza.model.get_units(self.application_name)
+        cmds = []
+        for check_name in self.nrpe_checks:
+            cmds.append(
+                'egrep -oh /usr/local.* /etc/nagios/nrpe.d/'
+                'check_{}.cfg'.format(check_name)
+            )
+        ret = self._retry_check_commands_on_units(cmds, units)
+        logging.info(ret)
```
Comment (fnordahl) [inline zaza/openstack/charm_tests/ovn/tests.py:75]: This produces a test output like this on success:
```
06:38:29 2020-10-01 06:38:29 [INFO] Confirm that the NRPE service check files are created.
06:38:29 2020-10-01 06:38:29 [INFO]  ... 
06:38:31 2020-10-01 06:38:31 [INFO] None
06:38:31 2020-10-01 06:38:31 [INFO] ok
```
And that may be a bit terse and/or confusing for someone not familiar with how the test works.

Would logging only when `ret` is not None be an alternative?

**Review by fnordahl — COMMENTED** [review]
This looks pretty good, have a comment in-line and awaiting a test result from the charms under test prior to approve/merge.

File: zaza/openstack/charm_tests/ovn/tests.py (hunk 1 above)
Comment (xavpaice) [reply zaza/openstack/charm_tests/ovn/tests.py]: I think that's an excellent plan.

**Review by fnordahl — APPROVED** [review]
LGTM, thanks!

**Pull request merged**